run.sh

```diff
@@ -8,25 +8,32 @@ set -euxo pipefail
 TARGET=${TARGET:-armv7-unknown-linux-gnueabihf}
 TARGET_SSH=${TARGET_SSH:-pi@raspberrypi.local}
 PROFILE=${PROFILE:-debug}
-RUN=${RUN:-1}
+RUN=${RUN:-0}
+DEPLOY=${DEPLOY:-1}
 SUFFIX=${SUFFIX:-}
 BIN=${BIN:-publish-mqtt}
 
-if [ $PROFILE = release ]
-then
-    time cross build --target $TARGET --release
-elif [ $PROFILE = debug ]
-then
-    time cross build --target $TARGET --bin $BIN
+if [ $PROFILE = release ]; then
+    PROFILE_FLAG=--release
+elif [ $PROFILE = debug ]; then
+    PROFILE_FLAG=''
 else
     echo "Invalid profile '$PROFILE'"
     exit 1
 fi
 
+time cross build $PROFILE_FLAG --target $TARGET --bin $BIN
+
 time rsync --progress target/$TARGET/$PROFILE/$BIN $TARGET_SSH:$BIN$SUFFIX
 
-if [ $RUN -eq 1 ]
-then
+if [ $DEPLOY = 1 ]; then
```

Comment:
DEPLOY seems like a weird name for this setting, when it really means 'restart the service and follow it'. If I saw the name without reading the code I would guess that DEPLOY controlled whether it rsyncs the binary across.